### ISSUE TITLE
Trouble building/deploying default app: 'cannot find module for page: /account'

### ISSUE THREAD

**Issue by elliott-king** [storefront-foundation/react-storefront#123]
# Bug report
I am following the [getting started](https://docs.reactstorefront.io/guides/getting_started) guide and, without changing code, deploying to [xdn](https://docs.reactstorefront.io/guides/xdn_deploying). `xdn build` and `xdn deploy` spit out the error:  'cannot find module for page: /account'.

A friend also tried this, same day, same problem.
## Steps to repro
1. I created a new storefront: `npm create react-storefront@^8.0.0 storefront-test`
 ```
✔ version … 1.0.0
✔ description … test react storefront
✔ repository url … 
✔ author … elliott-king
✔ license … UNLICENSED
✔ private … _false_ / true
✔ Will you be deploying your app on the Moovweb XDN? … no / _yes_
✔ Create a directory for this app? … no / _yes_
```
2. I tested the app with `npm start` - seems to be working fine.
3. I installed xdn with `npm i -g @xdn/cli` and intialized with `xdn init` - both ran fine.
4. `xdn deploy` cancels with the above error

## Expected behavior

I expect it to deploy correctly & the XDN UI should have a green dot instead of red.

## System information

- OS: macOS 10.15.6 and 10.15.5
- Version of React Storefront: "^8.13.0"
- Version of @xdn/cli 1.44.0

**Comment by markbrocato** [comment]
Looks like it may be an issue with packaging next 9.5 apps with the XDN.  We are investigating. Downgrading to next 9.4 should fix things.

**Comment by markbrocato** [comment]
It was indeed due to a change in Next.js 9.5. This has been fixed in @xdn/core@1.45.1.  You can upgrade your XDN libs by running: `xdn use latest`.
